### ISSUE TITLE
Search bar in Settings tab

### ISSUE THREAD

**Issue by devignx** [Eugeny/tabby#7055]
## Search
We will really appreciate a search feature in the settings tab
There are a whole bunch of customizations & other settings in the app, But we just lack a **search bar** to navigate directly to the preferred feature/option quicker
This will enhance the UX of the product

## Open A Terminal Directly
You can also include a toggle in the settings to **Open a preferred/default terminal** directly while opening the app
& Make the starting time a bit quicker than it is now

**Comment by Eugeny** [comment]
Re no. 2: this option already exists in Settings -> Terminal, although I presume it would have been easier to find with a search field ;) 
<img width="238" alt="image" src="https://user-images.githubusercontent.com/161476/186721149-2080a435-d2ab-41f6-b2d9-812376dd5194.png">

**Comment by devignx** [comment]
exactly, We just lack a Search field. Hope will include soon. I love this product btw, A tonne of customizations...

**Comment by dnut** [comment]
Yeah I've spent a lot of time clicking randomly through the settings because I had no idea where to look. It's tedious. A global settings search would be ideal, something like vscode.
